README.md

```diff
@@ -174,15 +174,15 @@ An IAM role session name that will be passed to each Cove session's
 
 `policy`: str
 
-A policy document that will be used as a session policy in each Cove session's
-`sts.assume_role()` call. Unless the value is None, it is passed through via the
-Policy parameter.
+A policy document that will be used as a session policy. A non-None value is
```

Comment:
Do we even need to explain this here? I feel like it's probably too much detail to discuss the precise interaction with the API call in here.

Reply:
I feel like it's niche enough that we really don't want to hide the real interface - I'd rather be explicit than make someone trawl the codebase then understand why we have to do the dict unpacking trick, etc

Reply:
Yes, fair enough. It's better to be explicit about how this interacts with boto3 as that's what people will already be familiar with. I find that I express it awkwardly here. Maybe in another PR we could make it explicit in a dedicated paragraph.